### ISSUE TITLE
EPIC: Create issues to Update Project Profile: VRMS

### ISSUE THREAD

**Issue by JackHaeg** [hackforla/website#7407]
### Overview
The project team from VRMS needs its project leadership variable information updated, so that the current team members can be properly displayed on the Hack for LA website

### Action Items
- [x] Review the Project Update Data below and then follow the instructions for either adding or deleting team members from the project's .md

<details><summary>Adding new volunteers to team page</summary>

- [x] Create a new issue using the [add new leader to project](https://github.com/hackforla/website/issues/new?assignees=&labels=good+first+issue%2C+P-Feature%3A+Project+Info+and+Page%2C+role%3A+back+end%2FdevOps%2C+role%3A+front+end%2C+size%3A+0.25pt%2C+time+sensitive&projects=&template=project-profile-leader-updates---add.md&title=Update+Project+Profile%3A+%5BProject+Name%5D+Add+%5BName+of+Volunteer%5D) template
- [x]  Customize the title with name of project and person to add
- [x]  Customize the body
   - [x]   In the ER, go to the value under the line "Project Card URL"
      - [x]  Copy the file name 
      - [x] On the line that starts "- [ ] In your IDE…" replace brackets with the copy (the text we are expecting is a file name with the extension .md)
   - [x] In the copy block below the line that starts, "Find the `leadership` variable and add the following profile."  add all the information from the ER for the person you are adding.
   - [x] On the checkbox line below, "- [ ] Verify the changes by viewing..." replace with name of project
   - [x] to Resource 2, find the project on the https://www.hackforla.org/projects/ page
      - [x] click on its link
      - [x] copy link
      - [x] replace brackets on resource 2 with the URL
   - [x] Add link to the ER on resource 3
- [x] Choose `Submit new issue` button
- [x] Add milestone `Onboarding flow`
- [x] Review, and if ready, add label
```
Ready for Prioritization
```
- [x] Add the URL to a comment on this ER
    - [ ] if this is the first add leader issue to be created, paste this into a comment and add the URL of the issue you just created.
```
### Adding Leader issues
-

### Complete
- [ ] Check this when all the Add leader issues have been made
```
- [x] if the above comment already exists, just add your new issue URL to the list. 
- [x] Check off the box, if it's the last one that needed to be created.
</details> 

<details><summary>Removing volunteers from team page</summary>

- [ ] Create a new issue using the [remove new leader to project](https://github.com/hackforla/website/issues/new?assignees=&labels=good+first+issue%2C+P-Feature%3A+Project+Info+and+Page%2C+role%3A+back+end%2FdevOps%2C+role%3A+front+end%2C+size%3A+0.25pt%2C+time+sensitive&projects=&template=project-profile-leader-updates---remove.md&title=Update+Project+Profile%3A+%5BName+of+Project%5D+Remove+%5BName+of+Volunteer%5D) template
- [ ] Customize the title with name of project and person to add
- [ ] Customize the body
   - [ ] In the ER, go to the value under the line "Project Card URL"
      - [ ] Copy the file name 
      - [ ] On the line that starts "- [ ] In your IDE…" replace brackets with the copy (the text we are expecting is a file name with the extension .md) 
   - [ ] Go to https://github.com/hackforla/website/tree/gh-pages/_projects and find and click on the file
      - [ ] click on code view
      - [ ] copy the code block
      - [ ] highlight the text on your new issue where it says `[Insert the lines of code to remove the specified leadership member]` and delete it and replace with the copy block
   - [ ] On the checkbox line below, "- [ ] Verify the changes by viewing..." replace with name of project
   - [ ]  to Resource 2, find the project on the https://www.hackforla.org/projects/ page
      - [ ] click on its link
      - [ ] copy link
      - [ ] replace brackets on resource 2 with the URL
   - [ ] Add link to the ER on resource 3
- [ ] Choose `Submit new issue` button
- [ ] Add milestone `Onboarding flow`
- [ ] Review, and if ready, add label

```
Ready for Prioritization
```
- [ ] Add the URL to a comment on this ER
   1. if this is the first remove leader issue to be created, paste this into a comment on this issue and add the URL of the issue you just created.
```
### Removing Leader issues
-

### Complete
- [ ] Check this when all the removal issues have been made
```
- [ ] if the above comment already exists, just add your new issue URL to the list. 
- [ ] Check off the box, if it's the last one that needed to be created.
</details> 

- [x] When you have completed making all the add and remove issues
   - [x] move this issue into the Questions/Review column
   - [x] add the `ready for product` label

### Project Update Data 

--- 

### Project Name

VRMS

### Project Card URL

vrms.md

### Update the Title?

NO

### Update the Description?

NO

### Update the Hero Image?

NO

### Update the Image?

NO

### Update the Technologies?

NO

### Update the Partner(s)?

NO

### Update the Tools?

NO

### Update GitHub Link?

NO

### Update Slack Link?

NO

### Update Test Site Link?

NO

### Update Demo Site Link?

NO

### Update Site Link?

NO

### Remove person(s) and info from Leadership?

YES

### If YES on remove person(s), edit the below to provide the information required. If NO, delete the template in the field below:

Name: 
- Judy Lee
- Alex Lee
- Amanda glover

### Add person(s) and info to Leadership?

YES

### If YES on add person(s), edit the below to provide the information required. If NO, delete the template in the field below:

Name: Jack Haeger
GitHub Handle: JackHaeg 
Role: Lead Product Manager
Slack Member ID: U059005TN1L

Name: Stephanie Samuels
GitHub Handle: Steph112792
Role: Product Manager
Slack Member ID: U03SJS7LYCC

Name: Katiuska Alecea de Leon
GitHub Handle: chukalicious
Role: Developer
Slack Member ID: U0100R46ESC

Name: Brad Morgan
GitHub Handle: bkmorgan3
Role: Developer
Slack Member ID: U779QRX3Q

Name: Evan Yang
GitHub Handle: evanyang1
Role: Developer
Slack Member ID: U02APE3V4Q3

Name: Angela Lee
GitHub Handle: awlfccamp
Role: Developer
Slack Member ID: U062NB2CKMZ

Name: Nikhil Trehan
GitHub Handle: ntrehan
Role: Developer
Slack Member ID: U078NLYGWRK

Name: Nora Zajzon
GitHub Handle: nora-zajzon
Role: Developer
Slack Member ID: U078JUC959C

Name: Mudassir Hussain
GitHub Handle: hussainmudassir
Role: Developer
Slack Member ID: U07GAGE89TJ

**Also, please change the following user's `GitHub Handle` & `Role`:** 
(the Name and Slack ID remain the same but he updated GitHub handle and Title)
- Name: Trillium Smith
- GitHub Handle: trillium 
- Role: Lead Developer
- Slack Member ID: U010G9864Q1

### Project Status

Active

**Comment by ExperimentsInHonesty** [comment]
Instructions for creating issues for 

<details><summary>removing leaders</summary>

1. Open a blank issue template
2. Add this title
```
Update Project Profile: [Insert project's name] Remove [Name of person]
```
3. Add this body
````
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/[Insert filename].md` file.
- [ ] Observe the existing syntax of the front matter block [^1] in the file.
- [ ]  Find the `leadership` variable and remove the following:
```
[Insert the lines of code to remove the specified leadership member]
```
- [ ] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [ ] [Insert name of project] page [^2]
- [ ] Once your pull request is merged, go to #[Insert ER number that initiated this issue] and
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move #[Insert ER number that initiated this issue] to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
- This issue was initiated by #[Insert ER number]
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: [Insert project specific page URL here]
````
4. Customize the links
5. Add labels: 
```
P-Feature: Project Info and Page
```
```
size: 0.25pt
```
```
role: front end
```
```
role: back end/devOps
```
```
good first issue
```
```
time sensitive
```
</details> 



<details><summary>Adding leaders</summary>

1. Open a blank issue template
2. Add this title
```
Update Project Profile: [Insert project's name] Add [Name of Person]
```
3. Add this body
````
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/[Insert filename].md` file.
- [ ] Observe the existing syntax of the front matter block [^1] in the file.
- [ ] Find the `leadership` variable and add the following profile. If there are other [Insert role]s, place [Insert leadership member's name]'s profile after the other [Insert role]s' profiles.
```
  - name: [Insert leadership member's name]
    github-handle: [Insert GitHub handle]
    role: [Insert role]
    links:
      slack: https://hackforla.slack.com/team/[Insert Slack member ID]
      github: https://github.com/[Insert GitHub handle]
    picture: https://avatars.githubusercontent.com/[Insert GitHub handle]
```
- [ ] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [ ] [Insert name of project] page [^2]
- [ ] Once your pull request is merged, go to #[Insert ER number that initiated this issue] and
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move #[Insert ER number that initiated this issue] to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
- This issue was initiated by #[Insert ER number]
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: [Insert project specific page URL here]
````
4. Customize the links
5. Add labels: 
```
P-Feature: Project Info and Page
```
```
size: 0.25pt
```
```
role: front end
```
```
role: back end/devOps
```
```
good first issue
```
```
time sensitive
```
</details>

**Comment by ExperimentsInHonesty** [comment]
### Removing Leader issues
- https://github.com/hackforla/website/issues/7471
- https://github.com/hackforla/website/issues/7470
- https://github.com/hackforla/website/issues/7469

### Complete
- [x] Check this when all the removal issues have been made

**Comment by ExperimentsInHonesty** [comment]
<details><summary>Adding leaders</summary>

1. Open a [blank issue template](https://github.com/hackforla/website/issues/new?assignees=&labels=Complexity%3A+Missing%2C+Feature+Missing%2C+role+missing%2C+size%3A+missing&projects=&template=blank-issue-template.md&title=)
2. Add this title
```
Update Project Profile: [Insert project's name] Add [Name of Person]
```
3. Customize the title
4. Use this body to replace the template body and then click submit

````
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/[Insert filename].md` file.
- [ ] Observe the existing syntax of the front matter block [^1] in the file.
- [ ] Find the `leadership` variable and add the following profile. 
```
  - name: [Insert leadership member's name]
    github-handle: [Insert GitHub handle]
    role: [Insert role]
    links:
      slack: https://hackforla.slack.com/team/[Insert Slack member ID]
      github: https://github.com/[Insert GitHub handle]
    picture: https://avatars.githubusercontent.com/[Insert GitHub handle]
```
- [ ] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [ ] [Insert name of project] page [^2]
- [ ] Once your pull request is merged, go to #[Insert ER number that initiated this issue] and
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move #[Insert ER number that initiated this issue] to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
- This issue was initiated by #[Insert ER number]
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: [Insert project specific page URL here]
````
5. Add labels: 
```
P-Feature: Project Info and Page
```
```
size: 0.25pt
```
```
role: front end
```
```
role: back end/devOps
```
```
good first issue
```
```
time sensitive
```
6. remove missing labels
7. Customize the links
    1.  Replace brackets with the ,md name of file
    2. After "Verify" section, replace with name of project
      1. to Resource 2, find the project on the https://www.hackforla.org/projects/ page
          1. click on its link
          2. copy link
          3. replace brackets on resource 2 with the URL
   3. Add link to the ER in the following places
      1. on the line that starts "Once your pull request is merged..."
      2. on the line that starts "If all the issues in the _Dependency..."
      3. under Resources
8.  Update comment button
9. Add milestone 3.02
10. Review, and if ready, add label
```
Ready for Prioritization
```
</details>

**Comment by ExperimentsInHonesty** [comment]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/[Insert filename].md` file.
- [ ] Observe the existing syntax of the front matter block [^1] in the file.
- [ ] Find the `leadership` variable and add the following profile. 
```
  - name: [Insert leadership member's name]
    github-handle: [Insert GitHub handle]
    role: [Insert role]
    links:
      slack: https://hackforla.slack.com/team/[Insert Slack member ID]
      github: https://github.com/[Insert GitHub handle]
    picture: https://avatars.githubusercontent.com/[Insert GitHub handle]
```
- [ ] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [ ] [Insert name of project] page [^2]
- [ ] Once your pull request is merged, go to the initiating ER [^3]
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move initiating ER [^3] to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: [Insert project specific page URL here]
[^3]: initiating ER:  #[Insert ER number]

**Comment by Rabia2219** [comment]
# New template issue draft to remove a single project member.

````
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] 
- [ ] 
- [ ] 

```
 
```
- [ ] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [ ] [Insert name of project] page [^2]
- [ ] Once your pull request is merged, go to the initiating ER [^3]
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move initiating ER [^3] to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: [Insert project specific page URL here]
[^3]: initiating ER:  #[Insert ER number]
````

**Comment by ExperimentsInHonesty** [comment]
### Adding Leader issues
- https://github.com/hackforla/website/issues/7473
- https://github.com/hackforla/website/issues/7486
- https://github.com/hackforla/website/issues/7844
- https://github.com/hackforla/website/issues/7849
- https://github.com/hackforla/website/issues/7850
- https://github.com/hackforla/website/issues/7878
- https://github.com/hackforla/website/issues/7877
- https://github.com/hackforla/website/issues/7876
- https://github.com/hackforla/website/issues/7875
- https://github.com/hackforla/website/issues/7879

### Complete
- [x] Check this when all the Add leader issues have been made

**Comment by HackforLABot** [comment]
Hi @mademarc, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by HackforLABot** [comment]
Hi @belunatic, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by belunatic** [comment]
i. Availability: Thu-Fri
ii. ETA: @daily at 12-3pm PST

**Comment by mademarc** [comment]
i Availability: 01/24/2025
ii.ETA: 01/31/2025

**Comment by ExperimentsInHonesty** [comment]
All issues are made and prioritized
